src/Runtime/Distributions/DiscreteChar.cs

```diff
@@ -1879,19 +1877,31 @@ internal sealed class StorageCache
             public static readonly Storage NonWordChar;
             public static readonly Storage Whitespace;
 
+            // Common distributions which can be produced by ToLower()
+            public static readonly Storage LowerOrDigit;
+            public static readonly Storage LowerWordCharOrDigit;
+            public static readonly Storage UpperComplement;
+
             private static readonly Storage[] PointMasses;
 
             static StorageCache()
             {
+                string LetterOrDigitsRanges(string baseRange) => baseRange + "09";
+                string WordCharRanges(string baseRange) => baseRange + "09__";
+
                 Uniform = Storage.CreateUncached(new CharRange[] { }, UniformProb, CharClasses.Uniform, UniformRegexRepresentation);
                 Digit = Storage.CreateUniformInRanges("09", CharClasses.Digit, DigitRegexRepresentation);
                 Lower = Storage.CreateUniformInRanges(LowerCaseCharacterRanges, CharClasses.Lower,LowerRegexRepresentation);
                 Upper = Storage.CreateUniformInRanges(UpperCaseCharacterRanges, CharClasses.Upper, UpperRegexRepresentation);
                 Letter = Storage.CreateUniformInRanges(LetterCharacterRanges, CharClasses.Letter, LetterRegexRepresentation);
-                LetterOrDigit = Storage.CreateUniformInRanges(LetterCharacterRanges + "09", CharClasses.LetterOrDigit, LetterOrDigitRegexRepresentation);
-                WordChar = Storage.CreateUniformInRanges(LetterCharacterRanges + "09__", CharClasses.WordChar, WordCharRegexRepresentation);
+                LetterOrDigit = Storage.CreateUniformInRanges(LetterOrDigitsRanges(LetterCharacterRanges), CharClasses.LetterOrDigit, LetterOrDigitRegexRepresentation);
+                WordChar = Storage.CreateUniformInRanges(WordCharRanges(LetterCharacterRanges), CharClasses.WordChar, WordCharRegexRepresentation);
                 NonWordChar = WordChar.Complement();
                 Whitespace = Storage.CreateUniformInRanges("\t\r  ", CharClasses.Unknown, null);
+
+                LowerOrDigit = Storage.CreateUniformInRanges(LetterOrDigitsRanges(LowerCaseCharacterRanges), CharClasses.Unknown, null);
```

Comment:
Why no symbol representation and regex representation?

Reply:
They are not strictly needed. Standard ToString() for DiscreteChar will create correct representations.

Reply:
What do you mean by 'correct' here? There may be some general way of dealing with this, but there is no helpful symbol defined for when you're looking at an automaton ToString().

Reply:
Ok, I'll come up with some symbol representation.
Is [↓#] fine for LoweOrDigit and ⮋ for LowerWordCharOrDigit?
Or propose something better :)

I used this list as a reference: http://xahlee.info/comp/unicode_arrows.html

Reply:
Recommend ⭽  and ⧬ respectively.

Reply:
Nice!